contrib/fwupd.spec.in

```diff
@@ -5,6 +5,7 @@
 %global libjcat_version 0.1.0
 %global systemd_version 231
 %global json_glib_version 1.1.1
+%global libfwupdplugin_lt_current 3
```

Comment:
changes to this file should probably be a separate commit

Reply:
Also, since this is "spec.in", can't this come from meson.build in a `configure()`?

Reply:
I guess, but I think it changes so little it's not worth the build pain.

Reply:
Actually, it's just 4 lines of pain. Changed, good idea.